### PR TITLE
chore(package): update `mime` v1.3.6...2.0.0

### PR DESCRIPTION
**Pull request by evilebottnawi**
**What kind of change does this PR introduce?**
Chore

**Did you add tests for your changes?**

not required

**If relevant, did you update the README?**

not required

**Summary**

Ref: https://github.com/broofa/node-mime/issues/168

**Does this PR introduce a breaking change?**

no

**Other information**

not required

### CODE REVIEW

**Comment by ryanjadhav** [comment]
+1 thanks for updating!

**Comment by evilebottnawi** [comment]
@michael-ciniawsky why `blocked`?

**Comment by johntron** [comment]
Just FYI in case you're unaware - NSP published [a security vulnerability in the mime package used by this project](https://nodesecurity.io/advisories/535). According to npm, this package gets 178,618 downloads a day. I'm sure many people can benefit from some expediency in merging this PR.

**Comment by michael-ciniawsky** [comment]
@evilebottnawi Requires [`node >= 6.0.0`](https://github.com/broofa/node-mime/blob/master/package.json#L8) 😞

**Comment by d3viant0ne** [comment]
Publish as a Major & drop NodeJS 4.x support?

**Comment by michael-ciniawsky** [comment]
@d3viant0ne `mime >=  v1.4.1` should be fixed [aswell](https://github.com/broofa/node-mime/issues/167#issuecomment-331975832)

**Comment by d3viant0ne** [comment]
Or that ^^

**Comment by d3viant0ne** [comment]
You will have to excuse me, i'm all for prodding people to use up to date Node versions when & wherever possible.

**Comment by evilebottnawi** [comment]
@michael-ciniawsky @d3viant0ne done

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   },
   "dependencies": {
     "loader-utils": "^1.0.2",
-    "mime": "^1.3.6",
+    "mime": ">=1.4.1",
```
Comment (michael-ciniawsky) [inline package.json]: `^1.4.1` 😛

**Pull request merged**